interfaces/external-modules/redux-saga.d.js

```diff
@@ -0,0 +1,485 @@
+    & (<T1, T2, T3, T4, R, Fn: Fn4<T1, T2, T3, T4, R>>(fn: Fn, t1: T1, t2: T2, t3: T3, t4: T4, ...rest: Array<void>) => SpawnEffect4<null, Fn, T1, T2, T3, T4>)
+    & (<T1, T2, T3, T4, T5, R, Fn: Fn5<T1, T2, T3, T4, T5, R>>(fn: Fn, t1: T1, t2: T2, t3: T3, t4: T4, t5: T5, ...rest: Array<void>) => SpawnEffect5<null, Fn, T1, T2, T3, T4, T5>)
+    & (<T1, T2, T3, T4, T5, T6, R, Fn: Fn6<T1, T2, T3, T4, T5, T6, R>>(fn: Fn, t1: T1, t2: T2, t3: T3, t4: T4, t5: T5, t6: T6, ...rest: Array<void>) => SpawnEffect6<null, Fn, T1, T2, T3, T4, T5, T6>)
+    & (<T, R, Fn: FnSpread<T, R>>(fn: Fn, t1: T, t2: T, t3: T, t4: T, t5: T, t6: T, ...args: Array<T>) => SpawnEffectSpread<null, Fn, T>);
+
+  /**
+   * SELECT STUFF
+   */
+  declare type SelectFn =
+    & (<Fn: SelectFn>(selector: Fn, ...rest: Array<void>) => SelectEffect0<Fn>)
+    & (<T1, Fn: SelectFn1<T1>>(selector: Fn, t1: T1, ...rest: Array<void>) => SelectEffect1<Fn, T1>)
+    & (<T1, T2, Fn: SelectFn2<T1, T2>>(selector: Fn, t1: T1, t2: T2, ...rest: Array<void>) => SelectEffect2<Fn, T1, T2>)
+    & (<T1, T2, T3, Fn: SelectFn3<T1, T2, T3>>(selector: Fn, t1: T1, t2: T2, t3: T3, ...rest: Array<void>) => SelectEffect3<Fn, T1, T2, T3>)
+    & (<T1, T2, T3, T4, Fn: SelectFn4<T1, T2, T3, T4>>(selector: Fn, t1: T1, t2: T2, t3: T3, t4: T4, ...rest: Array<void>) => SelectEffect4<Fn, T1, T2, T3, T4>)
+    & (<T1, T2, T3, T4, T5, Fn: SelectFn5<T1, T2, T3, T4, T5>>(selector: Fn, t1: T1, t2: T2, t3: T3, t4: T4, t5: T5, ...rest: Array<void>) => SelectEffect5<Fn, T1, T2, T3, T4, T5>)
+    & (<T1, T2, T3, T4, T5, T6, Fn: SelectFn6<T1, T2, T3, T4, T5, T6>>(selector: Fn, t1: T1, t2: T2, t3: T3, t4: T4, t5: T5, t6: T6, ...rest: Array<void>) => SelectEffect6<Fn, T1, T2, T3, T4, T5, T6>)
+    & (<T, Fn: SelectFnSpread<T>>(selector: Fn, t1: T, t2: T, t3: T, t4: T, t5: T, t6: T, ...rest: Array<T>) => SelectEffectSpread<Fn, T>)
+
+  declare type TakeFn = {
+    <P: Pattern>(                  pattern: P): TakeEffect<P>;
+    <P: Pattern>(channel: Channel, pattern: P): TakeEffect<P>;
+  }
+
+  declare type PutFn = {
+    <T: Object>(action: T): PutEffect<T>;
+    <T: Object>(channel: Channel, action: T): PutEffect<T>;
+  }
+
+  declare type ActionChannelFn = {
+    <P: Pattern>(pattern: P, buffer?: Buffer): ActionChannelEffect<P>;
+  }
+
+  declare type JoinFn = {
+    (task: Task): JoinEffect;
+  }
+
+  declare type CancelFn = {
+    (task: Task): CancelEffect;
+  }
+
+  declare type RaceFn = {
+    <T: {[key: string]: IOEffect}>(effects: T): RaceEffect<T>;
+  }
+
+  declare module.exports: {
+    take: TakeFn,
+    takem: TakeFn,
+    put: PutFn,
+    call: CallFn,
+    apply: ApplyFn,
+    cps: CpsFn,
+    fork: ForkFn,
+    spawn: SpawnFn,
+    join: JoinFn,
+    cancel: CancelFn,
+    select: SelectFn,
+    actionChannel: ActionChannelFn,
+    race: RaceFn,
+    cancelled: () => CancelledEffect,
+  }
+}
```

Comment:
It's published on flow-typed https://github.com/flowtype/flow-typed but it basically copies the file locally so you can version it, so that's the same